packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/request-body.test.ts

```diff
@@ -8,90 +8,95 @@ import {expect} from '@loopback/testlab';
 import {model, property} from '@loopback/repository';
 
 describe('requestBody decorator', () => {
-  describe('@requestBody', () => {
-    context('can build a correct "RequestBody" spec and', () => {
-      it('persists "description" and "required" into the generated schema', () => {
-        const requestSpec = {
-          description: 'A sample request body',
-          required: true,
-        };
-        class MyController {
-          @post('/greeting')
-          greet(@requestBody(requestSpec) name: string) {}
-        }
+  context('can build a correct "RequestBody" spec and', () => {
+    it('persists "description" and "required" into the generated schema', () => {
+      const requestSpec = {
+        description: 'A sample request body',
+        required: true,
+      };
+      class MyController {
+        @post('/greeting')
+        greet(@requestBody(requestSpec) name: string) {}
+      }
 
-        const r = getControllerSpec(MyController).paths['/greeting']['post']
-          .requestBody;
-        expect(r.description).to.eql('A sample request body');
-        expect(r.required).to.eql(true);
+      const r = getControllerSpec(MyController).paths['/greeting']['post']
```

Comment:
Can we use a more descriptive property name than `r`, for example `requestBody` or `body`?

The same comment applies to other places in this test file using `r` too.

Reply:
You are using `requestBodySpec` below, it would be great to use the same name everywhere for consistency.